src/components/Restoration/YourCompanySummary.vue

```diff
@@ -80,7 +80,7 @@ export default {
     }
   },
   computed: {
-    ...mapGetters([
+    ...mapState(useStore, [
```

Comment:
Are we going from using getters to reading state directly?

Reply:
![image](https://user-images.githubusercontent.com/3484109/231222939-8e6450ca-af69-48a8-9ac8-ece59faf4302.png)

Reply:
![image](https://user-images.githubusercontent.com/3484109/231223244-58e21e4e-08d2-4b97-9fc9-f77063919795.png)

Reply:
Not sure about mapWritableState yet..

Reply:
Well, you replaced all `@Mutation`, which is as close to writing directly to the state as anything, with actions that write directly to the state. I think this is fine -- we still go through store methods to change its state.

What I don't want to see is this (pseudo-code):
```
@State myVariable: string

this.myVariable = "hi there"
```